coreservices/partsrelationshipservice/terraform/modules/eventhub/outputs.tf

```
@@ -0,0 +1,16 @@
+output "eventhub_name" {
+  value       = azurerm_eventhub.main.name
+  description = "The event hub name."
+}
+
+output "send_primary_connection_string" {
+  value       = azurerm_eventhub_authorization_rule.send.primary_connection_string
+  description = "The primary connection string to send events."
+  sensitive   = true
+}
+
+output "listen_primary_connection_string" {
+  value       = azurerm_eventhub_authorization_rule.listen.primary_connection_string
+  description = "The primary connection string to receive events."
```

nit: maybe we can listen rather than receive. as other places.

```suggestion
  description = "The primary connection string to listen events."
```

Just a suggestion so up to you to decide.

---

aligned on "receive"